sympy/integrals/rubi/constraint.py

```diff
@@ -0,0 +1,79 @@
+from sympy.external import import_module
+matchpy = import_module("matchpy")
+
+if matchpy:
+    Constraint, substitute = matchpy.Constraint, matchpy.substitute
+else:
+    raise ImportError('MatchPy could not be imported')
```

Comment:
I can use `sys.exit()`(as done [here](https://github.com/sympy/sympy/blob/9ab4041226f8f472d278f393b92469cd4926ecee/bin/mailmap_update.py#L13)) but that will stop the whole test itself.